backend/src/routes/gamerRoutes.ts

```diff
@@ -140,6 +140,19 @@ router.post('/post', async (req: Request, res: Response) => {
   }
 });
 
+router.post('/post/:id/', async (req: Request, res: Response) => {
```

Comment:
Do not forget to use the authentication middleware on this router (in server.ts) in order to ensure that only authenticated user can use this route

Reply:
More globally, split this "big" router file in sub-router files for each action (for example, a router for post, for follow, ...)

Reply:
Sure, I'll correct it next commit, thanks